src/JustEat.StatsD/Buffered/StatsDMessageKind.cs

```diff
@@ -0,0 +1,9 @@
+namespace JustEat.StatsD.Buffered
+{
+    internal enum StatsDMessageKind
+    {
+        Counter,
+        Timing,
+        Gauge
```

Comment:
Should add a `,` to the end of the line so that adding a new member in the future is a 1 line diff, not 2.